src/modules/visits/queries/pending-visit-count-for-user-by-slug.graphql

```diff
@@ -0,0 +1,16 @@
+query PendingVisitCountForUserBySlug($user: uuid = "", $slug: String = "") {
```

Comment:
Avoid default values in queries since graphql won't notify us of missing variables when you do this.
It is usually better to have errors when something is missing, unless this was intended.

```suggestion
query PendingVisitCountForUserBySlug($user: uuid, $slug: String) {
```